azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/utils/TextUtils.java

```diff
@@ -25,4 +25,8 @@ public static String yellow(String message) {
     public static String green(String message) {
```

Comment:
why not make the color as a parameter? And refactor other methods to `showWarningText` and `showConfirmText`

Reply:
applyColorToText has done so

Reply:
This method only append the color to the text, so 'showWarningText` is not proper.